037-kafka-canary-roadmap.md

```diff
@@ -0,0 +1,106 @@
+# Kafka Canary roadmap
+
+This proposal describes a possible future set of improvement and features for the [Strimzi Canary](https://github.com/strimzi/strimzi-canary)
+
+## Current situation
+
+The project [README.md](https://github.com/strimzi/strimzi-canary/blob/main/README.md) explains the current feature set.
+
+The Canary was added to Strimzi under [proposal 13](013-kafka-canary.md).
+The implementation is in golang and uses the [Sarama](https://github.com/Shopify/sarama/) Kafka client.
```

Comment:
Just a nit ... here and in other places ... I think the usual use is Golang with capital `G`